optaweb-vehicle-routing-docs/src/main/asciidoc/run-locally.adoc

```diff
@@ -11,7 +11,11 @@ The script will:
 * Build the project if the standalone JAR file does not exist.
 * Launch OptaWeb Vehicle Routing by taking a single region argument or by selecting the region interactively.
 
-== Getting started
+== Quickstart mode
+
+In the quickstart mode, the script will take care of downloading the region that is needed to work with the built-in data set.
```

Comment:
In quickstart mode, the script downloads the region that is required to work with...